src/openrct2/ride/ride.h

```diff
@@ -125,14 +125,9 @@ typedef struct rct_ride_entry {
     uint8 category[RCT2_MAX_CATEGORIES_PER_RIDE];       // 0x1BE
     uint8 shop_item;                                    // 0x1C0
     uint8 shop_item_secondary;                          // 0x1C1
-#if defined(NO_RCT2)
     rct_string_id capacity;
     rct_string_id vehicleName;
-#endif
 } rct_ride_entry;
```

Comment:
If this structure no longer needs its size checked, remove it out of packing pragmas

Reply:
It does still need to be packed as it is still used in testpaint.